devicetypes/sky-nie/min-smart-plug.src/min-smart-plug.groovy

```diff
@@ -116,61 +174,44 @@ def executeConfigureCmds() {
 
     def cmds = []
 
-    if (!device.currentValue("switch")) {
-        cmds << switchBinaryGetCmd()
-    }
-
-    if (state.resyncAll || !device.currentValue("firmwareVersion")) {
-        cmds << secureCmd(zwave.versionV1.versionGet())
-    }
-
     configParams.each { param ->
         def storedVal = getParamStoredValue(param.num)
         def paramVal = param.value
-
         if (state.resyncAll || ("${storedVal}" != "${paramVal}")) {
-            logDebug "Changing ${param.name}(#${param.num}) from ${storedVal} to ${paramVal}"
             cmds << secureCmd(zwave.configurationV1.configurationSet(parameterNumber: param.num, size: param.size, scaledConfigurationValue: paramVal))
             cmds << secureCmd(zwave.configurationV1.configurationGet(parameterNumber: param.num))
         }
     }
 
     state.resyncAll = false
     if (cmds) {
-        sendCommands(delayBetween(cmds, 500))
+        sendHubCommand(cmds, 500)
```

Comment:
You don't have to call `sendHubCommand ` here. It should be enough to simply `return delayBetween(cmds, 500)`
This remark applies to: `configure(), ping(), on(), off(), refresh()`